### PR TITLE
Enhance ui-elements to be more accessible on smaller screens

### PR DESCRIPTION
**Pull request by Phil-Ah**
## Proposed changes
Enable breaking of content to be fully available on small screens or highly zoomed displays. This should help improve accessibility. 

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Please ensure the "what" and "why" are explained properly in this description as it will be used as the commit description on squashing your pull request commits. 

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [ ] New code is covered with unit tests
- [x] New ui components are tested inside the storybook (module ui-components only) 
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [ ] Feature has been tested with different permissions
- [ ] Documentation updated (only necessary for new features or changed behaviour)

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: scm-ui/ui-webapp/src/repos/permissions/components/PermissionsTable.tsx [hunk 1]
```diff
@@ -35,6 +36,15 @@ type Props = {
   namespaceOrRepository: Namespace | Repository;
 };
 
+const BreakingPermissionsTable = styled.table`
```
Comment (Pilopa) [inline scm-ui/ui-webapp/src/repos/permissions/components/PermissionsTable.tsx:39]: This styled component is unused.

File: scm-ui/ui-webapp/src/users/components/table/Details.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@
 import React from "react";
 import { WithTranslation, withTranslation } from "react-i18next";
 import { User } from "@scm-manager/ui-types";
-import { Checkbox, DateFromNow, MailLink, createAttributesForTesting } from "@scm-manager/ui-components";
+import {Checkbox, DateFromNow, MailLink, createAttributesForTesting, InfoTable} from "@scm-manager/ui-components";
```
Comment (Pilopa) [inline scm-ui/ui-webapp/src/users/components/table/Details.tsx:27]: Is prettier not configured ?

**Review by Pilopa — CHANGES_REQUESTED** [review]

File: scm-ui/ui-components/src/forms/AddKeyValueEntryToTableField.tsx [hunk 3]
```diff
@@ -38,12 +39,19 @@ type Props = {
   validateEntry?: (p: string) => boolean;
 };
 
-const FullWidthInputField = styled(InputField)`
-  width: 100%;
+const MobileWrapedDiv = styled.div`
```
Comment (Pilopa) [inline scm-ui/ui-components/src/forms/AddKeyValueEntryToTableField.tsx:42]: Wrapped ?

File: scm-ui/ui-components/src/forms/AddKeyValueEntryToTableField.tsx [hunk 4]
```diff
@@ -38,12 +39,19 @@ type Props = {
...
+  @media screen and (min-width: ${devices.tablet.width}px) {
+    & > fieldset {
```
Comment (Pilopa) [inline scm-ui/ui-components/src/forms/AddKeyValueEntryToTableField.tsx:44]: This selector is dependent on the internal structure of the InputField component. If we would keep the `FullWidthInputField` as a styled component, we could reference it here so no reference to the `fieldset` would be necessary.

**Comment by Pilopa** [comment]
- [ ] In the storybook story for breadcrumbs with long path names, the copy button is outside the viewport on mobile devices. This might not be a problem in the real-world application but it looks odd that the example breaks

File: scm-ui/ui-components/src/table/Table.tsx [hunk 5]
```diff
@@ -61,7 +61,7 @@ const Table: FC<Props> = ({ data, sortable, children, emptyMessage, className })
     return (
       <tr key={rowIndex}>
         {React.Children.map(children, (child, columnIndex) => {
-          return <td>{React.cloneElement(child, { ...child.props, columnIndex, row })}</td>;
+          return <td className={child.props.className}>{React.cloneElement(child, { ...child.props, columnIndex, row })}</td>;
```
[the remark this answers is not in the document]
Comment (fscholdei) [reply scm-ui/ui-components/src/table/Table.tsx:64]: The styling is applied to the entire column, as shown in the table stories. The child of a td is not affected.

**Review by fscholdei — DISMISSED** [review]

**Review by Pilopa — APPROVED** [review]

**Pull request merged**